source/agora/consensus/SlashPolicy.d

```diff
@@ -160,10 +152,7 @@ public class SlashPolicy
     {
         Hash[] keys;
         if (!this.enroll_man.getEnrolledUTXOs(keys) || keys.length == 0)
-        {
-            log.fatal("Could not retrieve enrollments / no enrollments found");
-            assert(0);
-        }
+            assert(0, "Could not retrieve enrollments / no enrollments found");
```

Comment:
Perhaps a const string could be used for this as it is used several times.

Reply:
For assert, it's quite nice to be able to just grep the message and find the line, so if it triggers, I'd recommend deduplicating it instead.

Reply:
Sure, saves an extra grep.